mlflow/R/mlflow/R/project-run.R

```diff
@@ -42,7 +41,7 @@
 #' @export
 mlflow_run <- function(uri = ".", entry_point = NULL, version = NULL, parameters = NULL,
                        experiment_id = NULL, experiment_name = NULL, backend = NULL, backend_config = NULL,
-                       no_conda = FALSE, storage_dir = NULL) {
+                       env_manager = NULL, storage_dir = NULL) {
```

Comment:
Nice! Separately, can we file a ticket to track removing conda from the R client install_mlflow() procedure entirely? I don't think there's a good reason to use conda just to install the MLflow Python client, and conda is problematic from a licensing perspective.

Reply:
For reference, see https://github.com/mlflow/mlflow/issues/6439

Reply:
Got it 👍

Reply:
Done. ML-24388.